Fix typo in README.md (#41)
jonspalmer/view_component_storybook: README.md

### Configuration

> By Default ViewComponent::Storybook expects to find stories in the folder `test/components/stories`. This can be configured but setting `stories_path` in `config/application.rb`. For example is you're using RSpec you might set the following configuration:

Reviewer: There are a few more typos here:

- "but setting" should be "by setting"
- "is you're" should be "if you're"

Reviewer 2: ```suggestion
By Default ViewComponent::Storybook expects to find stories in the folder `test/components/stories`. This can be configured by setting `stories_path` in `config/application.rb`. For example if you're using RSpec you might set the following configuration:
```
